crypto/asn1/a_mbstr.c

```diff
@@ -272,12 +270,15 @@ static int out_utf8(unsigned long value, void *arg)
 static int type_str(unsigned long value, void *arg)
 {
     unsigned long types;
+    const int native = ossl_fromascii(value);
```

Comment:
> Look in 6.2.5 paragraph 6.

This says that the storage sizes are the same, however 6.2.6.2 allows padding bits in either and it doesn't specify that there are the same number in each.  It would be permissible for an _int_ to have more padding bits than an _unsigned int_.  The statement of _M≤N_ rather than _M=N or M=N-1_ in paragraph 2 seems to allow this.

I love complex standards 🙄